src/application/controllers/installation.php

```diff
@@ -94,7 +94,7 @@ public function ajax_install() {
             $this->settings_model->set_setting('company_email', $company['company_email']);
             $this->settings_model->set_setting('company_link', $company['company_link']);
 
-            echo json_encode(AJAX_SUCCESS);
+            echo AJAX_SUCCESS;
```

Comment:
You either need to remove the `json_encode` or change the javascript AJAX_SUCCESS to, literally, `"SUCCESS"`.